extensions/hellonext-wallpapers/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "hellonext-wallpapers",
+  "title": "Hellonext Wallpapers",
+  "description": "Download and set Hellonext's exclusive wallpapers.",
+  "icon": "icon.png",
+  "author": "karthik",
+  "categories": [
+    "System",
+    "Other"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "set-hellonext-wallpaper",
+      "title": "Set Hellonext Wallpaper",
+      "description": "Set a wallpaper from Hellonext as your desktop wallpaper.",
+      "mode": "view"
+    },
+    {
+      "name": "random-hellonext-wallpaper",
+      "title": "Random Hellonext Wallpaper",
+      "description": "Set a random wallpaper from Hellonext as your desktop wallpaper.",
+      "mode": "view"
```

Comment:
Could be a [no-view](https://developers.raycast.com/information/manifest#command-properties) instead, and use a HUD to show a success message instead.